doc/website/getting-started/installation.md

```diff
@@ -67,8 +67,7 @@ We provide generic QNX SDP 7.0 toolchain files for ARM_64 and X86_64 in `./tools
 ARM_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_aarch64le.cmake`<br>
 X86_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_x86_64.cmake`
 
-!!! attention
-    Please make sure the directory `/var/lock` is created by your QNX image
+!!! attention Please ensure the filesystem on `/tmp` supports file locking.
```

Comment:
hmm, previously the `!!! attention` was in its own line. @mossmaurice is this necessary with mkdocs?

Reply:
it seems it's necessary https://squidfunk.github.io/mkdocs-material/reference/admonitions/